xptifw/CMakeLists.txt

```diff
@@ -14,17 +17,25 @@ if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
   set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Build type (default Release)" FORCE)
 endif()
 
-project (xptifw)
+if(MSVC)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc")
```

Comment:
Please add a comment why building with /EHsc

Reply:
It is needed because of the change: "printf" -> "std::cout <<"

Reply:
can you elaborate even more and add this as a source code comment?

Reply:
Is it ok to do so in separate PR?

Reply:
Yes, sure. I approved this PR